Wrapping/Generators/Python/Tests/extras.py

```diff
@@ -493,6 +503,21 @@ def custom_callback(name, progress):
     assert np.allclose(arr.transpose(), itk.array_view_from_image(image))
     assert np.allclose(arr.shape[::-1], itk.array_view_from_image(image).shape)
 
+    data_array = xr.DataArray(arr, dims=['q','x','y'])
+    try:
+        image = itk.image_from_xarray(data_array)
+        assert False
```

Comment:
debugging?

Reply:
This is intentional -- the code should not reach the `assert False` and raise an exception.

Reply:
Perhaps add a comment: `q is an unsupported dimension, so this should not be reached due to exception on the previous line`